app.js

```diff
@@ -0,0 +1,123 @@
+class ToDoList {
+    constructor() {
+        this.toDoTasks = [
```

Comment:
Podejrzewam, że zostawiłeś to sobie, żeby pamiętać jak wyglądają obiekty zadań, polecam do tego JSdocsa lub po prostu nauczyć się TypeScripta (szczerze mówiąc do tego mocno nakłaniam, mega działa na wyobraźnię przy rozmowach kwalifikacyjnych), ktory na Tobie wymusi użycie interfejsu ktory jest mega pomocny w czymś takim.